fdbserver/RestoreApplier.actor.cpp

```diff
@@ -84,13 +84,13 @@ ACTOR Future<Void> restoreApplierCore(RestoreApplierInterface applierInterf, int
 					updateProcessStatsTimer = delay(SERVER_KNOBS->FASTRESTORE_UPDATE_PROCESS_STATS_INTERVAL);
 				}
 				when(wait(exitRole)) {
-					TraceEvent("FastRestore").detail("RestoreApplierCore", "ExitRole").detail("NodeID", self->id());
+					TraceEvent("RestoreApplierCoreExitRole").detail("NodeID", self->id());
 					break;
 				}
 			}
 		} catch (Error& e) {
-			TraceEvent(SevWarn, "FastRestore")
-			    .detail("RestoreLoaderError", e.what())
+			TraceEvent(SevWarn, "FastRestoreApplierError", self->id())
+			    .detail("Error", e.what())
```

Comment:
```suggestion
			    .error(e)
```
This will give you error code as well.